NEWS

```diff
@@ -1,6 +1,62 @@
 Release a.b
 -----------
 
+New work and changes:
+
+ * samtools import now outputs query grouped to the header.
+   (PR#1633, thanks to Nils Homer)
+
+ * Made samtools view read error messages more generic.  Former error message
+   would claim that there was a "truncated file or corrupt BAM index file" with
+   no real justification.  Also reset errno in stream_view which could lead to
+   confusing error messages.
+   (PR#1645, addresses some of the issues in #1640.  Reported by Jian-Guo Zhou)
+
+ * Add bedcov option -c to report read count.
+   (PR#1644, fixes #1629.  Reported by Natchaphon Rajudom)
+
+ * Make samtools view -p also clear mqual, tlen and cigar.
+   (PR#1647, fixes #1606.  Reported by eboyden)
```

Comment:
Perhaps move this up adjace to the other `samtools view` change so things are clustered more by sub-command.